web/pandas/about/governance.md

```diff
@@ -49,145 +49,110 @@ manage project donations and acts as a parent legal entity. NumFOCUS is the
-array of backgrounds, viewpoints and talents in our team, which is why we
-explicitly do not define code as the sole metric on which Core Team membership
-will be evaluated.
-
-If a Core Team member becomes inactive in the project for a period of one year,
-they will be considered for removal from the Core Team. Before removal,
-inactive Member will be approached by the BDFL to see if they plan on returning
-to active participation. If not they will be removed immediately upon a Core
-Team vote. If they plan on returning to active participation soon, they will be
-given a grace period of one year. If they don’t return to active participation
-within that time period they will be removed by vote of the Core Team without
-further grace period. All former Core Team members can be considered for
-membership again at any time in the future, like any other Project Contributor.
-Retired Core Team members will be listed on the project website, acknowledging
-the period during which they were active in the Core Team.
-
-The Core Team reserves the right to eject current Members, other than the BDFL,
-if they are deemed to be actively harmful to the project’s well-being, and
-attempts at communication and conflict resolution have failed.
-
-### Conflict of interest
-
-It is expected that the BDFL and Core Team Members will be employed at a wide
-range of companies, universities and non-profit organizations. Because of this,
-it is possible that Members will have conflict of interests. Such conflict of
-interests include, but are not limited to:
-
--   Financial interests, such as investments, employment or contracting work,
-    outside of The Project that may influence their work on The Project.
--   Access to proprietary information of their employer that could potentially
-    leak into their work with the Project.
-
-All members of the Core Team, BDFL included, shall disclose to the rest of the
-Core Team any conflict of interest they may have. Members with a conflict of
-interest in a particular issue may participate in Core Team discussions on that
-issue, but must recuse themselves from voting on the issue. If the BDFL has
-recused his/herself for a particular decision, they will appoint a substitute
-BDFL for that decision.
-
-### Private communications of the Core Team
+## Community members
+
+The pandas community is composed of a diverse group of stakeholders, such as
+developers, contributors, individual and corporate users, etc. There are
+some groups which have specific responsibilities. We list them next.
+
+### Active maintainers
+
+Active maintainers (aka the core developer team) are contributors of the
+project who made significant contributions in the form of code, reviews,
+software design, documentation etc. Their role in the project is to
+advance the pandas software and goals.
+
+**Membership**: Contributors to the pandas project become maintainers after
+showing significant contributions over a period of one year, and with
```

Comment:
Maybe good to note that typically the contributor is nominated by an existing active member before consensus is decided